driver/src/driver/scheduler.rs

```diff
@@ -293,20 +316,21 @@ mod tests {
         let mut driver = MockStableXDriver::new();
         driver
             .expect_run()
-            .with(eq(U256::from(0)))
-            .returning(|_| DriverResult::Ok);
+            .withf(|b, t| *b == U256::from(0) && *t == Duration::from_secs(20))
```

Comment:
why not use eq here?

Reply:
I couldn't find a way to use `eq` with more than one parameter. Maybe I'm missing something obvious.

Reply:
It should be `with(eq(U256...), eq(Duration...))`

Reply:
That was easy... Thanks!